pkg/image/api/helper.go

```diff
@@ -128,6 +128,18 @@ func (r DockerImageReference) AsRepository() DockerImageReference {
 	return r
 }
 
+// DaemonMinimal clears defaults that Docker assumes.
```

Comment:
seems safer to leave it explicitly specified, in case these default assumptions change someday?  also can't i configure my dockerdaemon such that "docker.io" is not the first searched location?